romancal/resample/resample_step.py

```diff
@@ -157,25 +157,17 @@ def process(self, input):
         result = resamp.resample_group(range(len(input_models)))
         result.meta.filename = output_filename
 
-        self._final_updates(result, input_models)
+        self._final_updates(result)
 
         return result
 
-    def _final_updates(self, model, input_models):
+    def _final_updates(self, model):
         model.meta.cal_step["resample"] = "COMPLETE"
 
-        # if pixel_scale exists, it will override pixel_scale_ratio.
-        # calculate the actual value of pixel_scale_ratio based on pixel_scale
-        # because source_catalog uses this value from the header.
-        model.meta.resample.pixel_scale_ratio = (
-            self.pixel_scale / np.sqrt(model.meta.photometry.pixel_area * SR_TO_ARCSEC2)
-            if self.pixel_scale
-            else self.pixel_scale_ratio
-        )
-        model.meta.resample.pixfrac = self.pixfrac
-        if model.meta.photometry.pixel_area is not None:
-            model.meta.photometry.pixel_area *= model.meta.resample.pixel_scale_ratio**2
-        model.meta.resample["good_bits"] = self.good_bits
```

Comment:
Just noting for myself: this is being done in the blending / finalize, so isn't needed here as well.